mobile/src/pages/game/game.ts

```diff
@@ -42,9 +44,9 @@ export class GamePage {
     return this.goals[player.id];
   }
 
-  playerGoal(playerId){
-    if(!this.isFinish()){
-      let player = _.find(this.players, {id: playerId});
+  playerGoal(playerId) {
+    if (!this.isFinish()) {
+      const player = _.find(this.players, (x) => x.id) as IPlayer;
```

Comment:
i think you should use `(x) => x.playerId` inestead of `x.id`

Reply:
Actually it should end up as `_.find(this.players, (x) => x.id === playerId);`. I've added explicite type cast to `IPlayer` due to misunderstaning TS typechecker error. `_.find` function accepts as a second argument iterratee function which return type is `boolean`, and in this case it got iterratee returning `number`. Fixed it within next commit.